internal/pkg/template/templates/workloads/services/backend/manifest.yml

```
@@ -43,18 +43,16 @@ count: {{.Count.Value}}       # Number of tasks that should be running in your s
 {{- if not .TaskConfig.IsWindows }}
 exec: true     # Enable running commands in your container.
 {{- end}}
-  
+{{- if .ImageConfig.Port}}
+network:
+  connect: true # Enable Service Connect for in-environment traffic between services.
```

nit: 
```suggestion
  connect: true # Enable Service Connect for intra-environment traffic between services.
```
and do we want to include a link to AWS docs?